### DISCUSSION TITLE
Which python kernel?

### DISCUSSION THREAD

**Discussion by BioNCphilic** [earthlab-education/Earth-Analytics-AY24#576] [Q&A] [answered]
The good news is that I removed all programs and re-installed git, bash, and conda. I have my repository cloned and am ready to go! My issue is deciding on the "correct" kernel. The [instructions](https://canvas.colorado.edu/courses/109233/files/76793536?module_item_id=5890577) for loading python on a PC at the very end d.iv - says not to use the base kernel. So here is a screenshot of my choices and I'm leaning toward Jupyter Notebook, but I'm hesitant to screw things up again. Any advice would be appreciated so i can work on this over the weekend.
![Screenshot 2024-10-24 155654](https://github.com/user-attachments/assets/b09eedfa-2e49-4ffe-96cd-dbd1bd1806db)

**Comment by akpost21** [comment]
Hi Randi, that's great! You can click on "select kernel" in the upper right of the screen and then some options should pop up in the search bar at the center top. Then select "Python Environments" and then "earth-analytics-python" should pop up as an option if you ran the "conda env create" command. See if that works.

**Reply by BioNCphilic** [reply]
The screenshot shows what I get when I try to select kernel - nothing pops up except a prompt to search. The screenshot shows my options.

**Reply by akpost21** [reply, marked as answer]
Hmm @eculler Do you have suggestions? @BioNCphilic It might be worth popping into office hours today if you can make it so Elsa can help you figure it out.
